data/src/main/java/com/teamagam/gimelgimel/data/geometry/repository/inMemory/InMemoryGeoDataCache.java

```diff
@@ -0,0 +1,72 @@
+package com.teamagam.gimelgimel.data.geometry.repository.inMemory;
+
+import com.teamagam.gimelgimel.domain.geometries.entities.GeoEntity;
+import com.teamagam.gimelgimel.domain.geometries.entities.VectorLayer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import rx.Observable;
+
+public class InMemoryGeoDataCache {
+    private List<VectorLayer> mVectorLayers;
```

Comment:
I don't think so, because we don't use a different data structure to store it.
We can create a class when necessary (Store the layers in the server).